card/mod/search/set/abstract/03_filter/quick_filters.haml

```diff
@@ -0,0 +1,14 @@
+.quick-filter._quick-filter
+  %i.fa.fa-bolt.text-muted.p-2.float-left
+  .quick-filter-links.d-flex.flex-wrap
+    - quick_filter_list.each do |hash|
+      - link_hash = JSON(hash)
+      - filter_key = hash.keys.first
+      - value = hash.values.first
+      %a{ "data-filter": link_hash,
+          href: "#",
+          class: "quick-filter-by-#{filter_key}",
+          title: "Quick Filter" }
+        = mapped_icon_tag filter_key
+        // FIXME: mapped_icon_tag is WikiRate concept
```

Comment:
do we want to make mapped_icon_tags a decko thing?  Or should I just put a method here that we can override.  (Leaning towards the second option).

Reply:
probably not worth the effort to make it a decko thing